### DISCUSSION TITLE
Vercel not deploying? is this an error?

### DISCUSSION THREAD

**Discussion by enyelsequeira** [vercel/vercel#7666] [Help]
Hello all, 

Since earlier today, I have not been able to deploy/build an app that I am working on, now I've checked the types and build the app locally, so I know it builds and the types are correct however when deploying to vercel I keep getting this error

I was looking online and there was someone else with this error, but I am wondering if this is  a vercel error? 

<img width="1059" alt="CleanShot 2022-04-11 at 11 46 56@2x" src="https://user-images.githubusercontent.com/50872387/162662133-fb0fff11-91bd-4fa6-9a5a-b281bc79ebf7.png">

**Comment by leerob** [comment]
Hey! This doesn't appear to be a Vercel error, but instead an error building your application. Does it build locally?

**Reply by enyelsequeira** [reply]
Hey Lee, so actually it does build locally, this is a pretty new issue from what I was able to see. 
Ignoring the TS build error seems to fix it after trying it out, but not sure if it’s the right way to do it   


https://github.com/vercel/next.js/issues/36019

**Reply by vithushan19** [reply]
Hi, I am having the same error. My application is building locally, but failing on Vercel. I tried re-deploying old versions that were passing last week, and they are now failing on Vercel. 
<img width="870" alt="Screen Shot 2022-04-11 at 3 50 23 PM" src="https://user-images.githubusercontent.com/4795012/162818384-e4f7a4f3-64a8-41fa-8884-1733e2679a46.png">

**Comment by faccodev** [comment]
Same error in my application, builds nomaly in local, but not in vercel.
![image](https://user-images.githubusercontent.com/19209970/162968210-2eb0b096-24d7-4a06-baa3-384b84842f0f.png)

**Comment by leerob** [comment]
Okay, I think I've figured this out. It seems to be a breaking change in `@types/react`. You need to pin a version of that package that works for you.

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56210

**Comment by faccodev** [comment]
in my case the issue as solved runing this on terminal:
`yarn upgrade @types/react`

and commited the changes in package.json to run vercel build
